gammapy/cube/make.py

```diff
@@ -209,6 +223,8 @@ def __init__(
         geom_true=None,
```

Comment:
There are basically 3 geoms involved here:

1. [e_reco, spatial_fine] - The geom for counts and background - `geom`
2. [e_true, spatial_coarse] - The geom for IRFs - `geom_true`
3. [e_true, spatial_fine] - The final exposure geom. This is created on the fly

Reply:
This makes sense. I guess alternatively one could change to the parametrisation `geom, energy_axis_true, rad_axis, migra_axis, bins_irf, margin_irf`. Which is a bit simpler, because users don't have to create two geometries. But I'm fine to keep as is...

Reply:
We can move to the parametrisation you suggest, specially when we have good defaults for the `e_true` and other axes. I think the user should normally not be interested in IRF binnings. Perhaps we keep this for now and change this once we remove `MapMaker`